src/dlsproto/client/request/internal/GetRange.d

```diff
@@ -272,6 +275,9 @@ private scope class GetRangeHandler
 
     private bool connect ( )
     {
+        if (this.request_started)
```

Comment:
Maybe add a comment here explaining why we don't restart the request.

Reply:
Updated with a comment.